src/components/Spinner.js

```diff
@@ -0,0 +1,23 @@
+export default function Spinner() {
+  return (
+    <div role="status">
+      <svg
+        aria-hidden="true"
+        class="w-8 h-8 mr-2 text-gray-200 animate-spin dark:text-gray-600 fill-red-600"
+        viewBox="0 0 100 101"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
```

Comment:
idk what a path is, but I'm assuming most of this would be like, we just give them this to copy & paste? Also what is a Spinner

Reply:
Yes this should 100% be provided to people. `path` is an SVG element (see: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/path). This is a loading spinner.

Reply:
This should just be included in the codespace template.